docs/streams/kafka_on_gdn/fluent-bit-client.md

```diff
@@ -0,0 +1,98 @@
+---
+sidebar_position: 60
+title: Fluent Bit Client
+---
+
+Fluent Bit is an open-source and multi-platform log processor tool that aims to be a generic Swiss knife for logs processing and distribution.
+
+Fluent Bit works as a logging pipeline chunked into logical parts like Inputs, Parsers, Filters and Outputs. Kafka client implementation able in Output part. It means Fluent bit produces records and sends to Kafka.
+
+Full Kafka plugin documentation can be found [here](https://docs.fluentbit.io/manual/pipeline/outputs/kafka)
+
+Kafka plugin is on top of `libdrkafka` [library](https://github.com/edenhill/librdkafka/).
+
+### Installation
+
+The correct installation was proven on Ubuntu OS. The bellow steps will show how to build and install Fluent Bit with correct plugins and modules.
+
+#### Prequisites
+
+Build Fluent Bit from scratch requires installing `git` and then clone a repository from [GitHub](https://github.com/fluent/fluent-bit) and checkout to the latest version.
+
+```
+sudo apt update
+sudo apt install git
+git clone https://github.com/fluent/fluent-bit.git
+git checkout 1.8.0-pack-fixes
+```
+
+Building of Fluent Bit requires other Ubuntu packages.
+Fluent Bit uses CMake as it build system. The suggested procedure to prepare the build system consists on the following steps:
+
+```
+sudo apt install -y build-essential
+sudo apt install -y make
+sudo apt install -y cmake
+sudo apt install -y libsasl2-dev
+sudo apt install -y bison
+sudo apt install -y flex
+sudo apt install -y libssl-dev
+```
+
+#### Consumer
+
+To verify that our message has indeed been produced to the topic, let's create a consumer to consume our message:
+
+```node
+const consumer = kafka.consumer({ groupId: '<my-group-id>' })
+
+await consumer.connect()
+await consumer.subscribe({ topic: '<c8globals or c8locals>.<my-topic>', fromBeginning: true })
+
+await consumer.run({
+  eachMessage: async ({ topic, partition, message }) => {
+    console.log({
+      value: message.value.toString(),
+    })
+  }
+})
+```
+
+### Build
+
+Change to the `build` directory inside the Fluent Bit sources and let CMake configure the project specifying where the root path is located with Kafka plugin and TLS module:
+
+```
+cd fluent-bit/build
+cmake -DFLB_OUT_KAFKA=On -DFLB_TLS=On ..
+make
+```
+
+### Run
```

Comment:
This topic reads differently than previous topics. Different tone, different content. Try to be consistent with the different topics in this folder as much as possible. This helps readers situate themselves in the content.

Reply:
Fluent Bit works differently than other clients, hence the documentation is different